frontend/src/App.scss

```diff
@@ -532,3 +532,9 @@ fill: #1B1B1B;
 .rdw-link-decorator-icon {
   display: none;
 }
+
+.resource-link-wrapper {
+    max-width: 100%;
+    overflow-wrap: break-word;
+    word-break: break-word;
+}
```

Comment:
this is fine, and I approved this previously, but a best practice would be to create a "break-word" class or something that isn't specific to resource-links; it's a better name and that way we are more likely to reuse it and not create another functionally identical class.